src/components/LoginForm/LoginForm.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
```

Comment:
We can safely remove import of React itself from our code, but it's optional.

Also I suggest to import useState hook at the top of file, not from React itself.

`import {useState} from 'react';`